webui/tests/package-lock.json

```diff
@@ -1419,6 +1427,12 @@
           "integrity": "sha512-EykJT/Q1KjTWctppgIAgfSO0tKVuZUjhgMr17kqTumMl6Afv3EISleU7qZUzoXDFTAHTDC4NOoG/ZxU3EvlMPQ==",
           "dev": true
         },
+        "lodash": {
+          "version": "4.17.15",
```

Comment:
this is  using a version less than 4.17.19
> │ Patched in    │ >=4.17.19                                                    │

so it shows up as a vulnerability. should we force resolution or update the responsible package?

Reply:
`"@cypress/webpack-preprocessor": "^5.4.1"` is the responsible package and they are still using `4.17.15`. using force resolution to resolve this

Reply:
I looked that up it seems they don't have a newer release